hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/KeyOutputStream.java

```diff
@@ -709,4 +709,23 @@ private void checkNotClosed() throws IOException {
               + blockOutputStreamEntryPool.getKeyName());
     }
   }
+
+  protected BlockOutputStreamEntryPool getBlockOutputStreamEntryPool() {
+    return blockOutputStreamEntryPool;
+  }
+
+  protected void setBlockOutputStreamEntryPool(
```

Comment:
we can use   @VisibleForTesting annotation as used in testcase

Reply:
sure!